test/unit/kernel/src/KernelThrow.cpp

```diff
@@ -0,0 +1,85 @@
+/* Copyright 2022  René Widera, Mehmet Yusufoglu, Andrea Bocci
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+#include <alpaka/core/RuntimeMacros.hpp>
+#include <alpaka/test/acc/TestAccs.hpp>
+#include <alpaka/test/queue/Queue.hpp>
+
+#include <catch2/catch_template_test_macros.hpp>
+#include <catch2/catch_test_macros.hpp>
+
+class KernelWithThrow
+{
+public:
+    template<typename TAcc>
+    ALPAKA_FN_ACC auto operator()(TAcc const& acc) const -> void
+    {
+        using Idx = alpaka::Idx<TAcc>;
+        using Dim = alpaka::Dim<TAcc>;
+        using Vec = alpaka::Vec<Dim, Idx>;
+        Vec const globalThreadIdx = alpaka::getIdx<alpaka::Grid, alpaka::Threads>(acc);
+        Vec const globalThreadExtent = alpaka::getWorkDiv<alpaka::Grid, alpaka::Threads>(acc);
+
+        // Map the three dimensional thread index into a one dimensional thread index space.
+        auto const linearizedGlobalThreadIdx = alpaka::mapIdx<1u>(globalThreadIdx, globalThreadExtent);
+
+        if(static_cast<unsigned>(linearizedGlobalThreadIdx[0u]) == 0u)
+        {
+            printf("linearizedGlobalThreadIdx: %lu\n", linearizedGlobalThreadIdx[0]);
```

Comment:
Windows CI fails here: 

```
 D:\a\alpaka\alpaka\test\unit\kernel\src\KernelThrow.cpp(29,20): error C2220: the following warning is treated as an error [D:\a\alpaka\alpaka\build\test\unit\kernel\kernelTest.vcxproj]
```

Reply:
solved by changing  integer literal suffix in  `printf` to `u` (from lu)  and Acc index type to `uint32_t` from `size_t` .